vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionSpaceOrderUtility.java

```diff
@@ -0,0 +1,92 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import edu.asu.diging.vspace.core.model.ExhibitionSpaceOrderMode;
+import edu.asu.diging.vspace.core.model.IExhibition;
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.model.impl.SpaceStatus;
+import edu.asu.diging.vspace.core.model.impl.SpacesCustomOrder;
+import edu.asu.diging.vspace.core.services.IExhibitionManager;
+import edu.asu.diging.vspace.core.services.IExhibitionSpaceOrderUtility;
+
+/**
+ * Utility class to order the spaces in the Exhibition using a user defined custom order.
+ * @author prachikharge
+ *
+ */
+@Component
+public class ExhibitionSpaceOrderUtility implements IExhibitionSpaceOrderUtility {
+    
+    @Autowired
+    private  IExhibitionManager exhibitionManager;
+    
+    /**
+     * This method sorts the spaces in alphabetical order.
```

Comment:
There should probably be a comment that this method orders spaces in place, so that the passed in list of spaces will be sorted after the method is done.